### ISSUE TITLE
Persistence not passed to Realtime Database

### ISSUE THREAD

**Issue by YousefED** [invertase/react-native-firebase#529]
According to https://rnfirebase.io/docs/v3.0.*/core/initialize-apps, I should be able to set the persistence flag to true in config to enable Realtime Database Persistence.

I realize this is not supported for the Default App, but I'm using fb.initializeApp(config, "name") to initialize a different app. I think this feature is still meant to work for non-default apps right?

Setting breakpoints in the code, I noticed that persistence: true, is never passed to the database, so "if (this._options.persistence) {" always evaluates to false (https://github.com/invertase/react-native-firebase/blob/master/lib/modules/database/index.js).

**Comment by Ehesp** [comment]
cc @Salakar

**Comment by Salakar** [comment]
@YousefED thanks for flagging, have pushed up a fix for this that will appear in the next release,

As a temporary workaround until it's published you can call the same function as it would internally if it's enabled:

```javascript
// call before any usage of the database - otherwise will throw an error
myNewApp.database()._native.setPersistence(true);
```

**Comment by YousefED** [comment]
Awesome, thanks!

**Comment by chrisbianca** [comment]
This has now been released as part of 3.0.5
